src/lib.rs

```diff
@@ -136,7 +136,13 @@
 )]
 
 #[macro_use]
-extern crate html5ever;
+pub extern crate html5ever;
+
+pub use cssparser;
+pub use ego_tree;
+pub use selectors;
```

Comment:
Doesn't this one make the more selective `pub use selectors::{attr::CaseSensititvy, Element};` below redundant?